### DISCUSSION TITLE
USB，以及邮箱缺少依赖

### DISCUSSION THREAD

**Discussion by Linux-qitong** [linuxdeepin/developer-center#4172] [Troubleshooting | 故障排查]
https://bbs.deepin.org/phone/zh/post/255513
新版邮箱不能更新提示缺依赖。
https://bbs.deepin.org/phone/post/255496
这是USB提示异常

**Comment by xzl01** [comment]
请一个帖子只反馈一个问题，另，如果是转载社区的提问贴，请将原帖内容复制过来，并且补充详细信息

**Reply by Linux-qitong** [reply]
好的，下次注意。
